### ISSUE TITLE
"First Moments" Analysis

### ISSUE THREAD

**Issue by AuroraLS3** [plan-player-analytics/Plan#2299]
### I would like to be able to..

.. compare impact of actions taken to improve new player retention.

- .. see the session activity of players who joined within the last 24h or last 7 days
- .. compare the session activity to that of previous weeks
- .. see how many players were online during the session
- .. see if the player is banned
- .. see how much the player played
- .. get a link to their player page

### Is your feature request related to a problem? Please describe.

- Current retention metrics are sub-par for measuring the impact of new player retention. It is difficult to see the difference in actions from 30 day insights and a single number.
- The current sessions view is too crowded to focus on new players
- The current server session calendar contains data for all players instead of just new players
- The query page does not give insight into session activity of the matched players

**Comment by AuroraLS3** [comment]
Quick mockup of this feature in paint
![image](https://user-images.githubusercontent.com/24460436/159153560-eeb1dd47-23bd-47ff-a178-9bbffdcf45d5.png)

**Comment by AuroraLS3** [comment]
Blocked by #2260

**Comment by AuroraLS3** [comment]
Created the mockup in React, continuing some other time since this seems to take a while.

![image](https://user-images.githubusercontent.com/24460436/230874661-53158cad-8048-41cf-bfd9-10b438b7be1c.png)

### TODO

- [ ] Create json endpoint
  - [ ] Input 
    - [ ] ISO-8601 Date
    - [ ] Server UUID (optional)
  - [ ] Output
    - [ ] List of sessions within that day (of players who registered that day)
    - [ ] Players online graph data within that day (Consider multi-proxy)
    - [ ] Player metadata such as Ban status
    - [ ] Color for online graph, sessions and new players
    - [ ] Formatted date for the ISO-8601 date that was input
- [ ] Make the mockup use the endpoint
- [ ] Wire up the previous and next buttons to change the date
- [ ] Cleanup and make the whole thing useable
- [ ] Consider if it would be interesting to see sessions like this on the Sessions tab and open new ticket for that

**Comment by AuroraLS3** [comment]
This doesn't unfortunately seem to be at all useful when using real world data, especially comparing to how useful clicking on a calendar day seems to be to this.

I'm scrapping this feature.
![image](https://github.com/plan-player-analytics/Plan/assets/24460436/2df0cf52-2dcf-4e2e-8156-d79c1ab37069)
